test/typecheck/main.go

```diff
@@ -266,7 +266,7 @@ func (c *collector) handlePath(path string, info os.FileInfo, err error) error {
 	}
 	if info.IsDir() {
 		// Ignore hidden directories (.git, .cache, etc)
-		if len(path) > 1 && path[0] == '.' {
+		if len(path) > 1 && (path[0] == '.' || path[0] == '_') {
```

Comment:
is this related to GOPATH or something else?

Reply:
similar but different - I happen to have an _thockin dir with little tools I occasionally use, including some Go code.  Tools should all ignore that.  Typecheck did not ignore it.

Reply:
This will ignore things like https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/code-generator/_examples as well, right? I guess that's a feature, not a bug.